### PR TITLE
add sylius 1.11

### PR DESCRIPTION
**Pull request by oallain**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Fixed issue   | 
| License       | MIT

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -83,25 +84,26 @@ jobs:
...
-                        php: 7.3
-                    -   sylius: 1.10.0
-                        symfony: 4.4
-
+                    - sylius: 1.9.0
+                      php: 8.0
+                    - sylius: 1.9.0
+                      php: 8.1
+                    - sylius: 1.10.0
+                      symfony: 4.4
```
Comment (Jibbarth) [inline .github/workflows/ci.yaml]: really ? 1.10 is still compatible with 4.4

File: Makefile [hunk 2]
```diff
@@ -28,30 +32,36 @@ phpunit: phpunit-configure phpunit-run ## Run PHPUnit
...
 sylius-standard:
-	${COMPOSER_ROOT} create-project sylius/sylius-standard ${TEST_DIRECTORY} "~${SYLIUS_VERSION}"
+	${COMPOSER_ROOT} create-project sylius/sylius-standard ${TEST_DIRECTORY} "~${SYLIUS_VERSION}" --no-install --no-scripts
+	${COMPOSER} require sylius/sylius:"~${SYLIUS_VERSION}" -n
+
+update-dependencies:
+	${COMPOSER} config extra.symfony.require "^${SYMFONY_VERSION}"
+	${COMPOSER} require --dev donatj/mock-webserver:^2.1 --no-scripts --no-update
+ifeq ($(shell [[ $(SYMFONY_VERSION) == 4.4 && $(PHP_VERSION) == 7.4 ]] && echo true ),true)
+	${COMPOSER} require sylius/admin-api-bundle:1.10 --no-scripts --no-update
```
Comment (Jibbarth) [inline Makefile]: even on sylius 1.9  ?

File: Makefile [hunk 3]
```diff
@@ -28,30 +32,36 @@ phpunit: phpunit-configure phpunit-run ## Run PHPUnit
...
+endif
+ifeq ($(SYLIUS_VERSION), 1.8.0)
```
Comment (Jibbarth) [inline Makefile]: You don't have this version in your matrix

File: composer.json [hunk 4]
```diff
@@ -5,49 +5,50 @@
...
+        "phpmd/phpmd": "2.12.0",
+        "phpro/grumphp": "1.13.0",
+        "phpspec/phpspec": "7.2.0",
         "phpstan/extension-installer": "^1.0",
-        "phpstan/phpstan": "^0.12.1",
-        "phpstan/phpstan-doctrine": "^0.12.1",
-        "phpstan/phpstan-phpunit": "^0.12.1",
-        "phpstan/phpstan-strict-rules": "^0.12.1",
-        "phpstan/phpstan-webmozart-assert": "^0.12.1",
+        "phpstan/phpstan": "1.7.15",
```
Comment (Jibbarth) [inline composer.json]: what about v1.8 ?

File: composer.json [hunk 5]
```diff
@@ -5,49 +5,50 @@
...
-        "sylius-labs/coding-standard": "^4.0",
-        "symfony/browser-kit": "^3.4|^4.3",
-        "symfony/debug-bundle": "^3.4|^4.3",
-        "symfony/dotenv": "^4.3",
-        "symfony/intl": "^3.4|^4.3",
-        "symfony/lock": "^3.4|^4.3",
-        "symfony/web-profiler-bundle": "^3.4|^4.3",
-        "symfony/web-server-bundle": "^3.4|^4.3",
-        "webmozart/assert": "^1.6"
+        "sensiolabs/security-checker": "v6.0.3",
```
Comment (Jibbarth) [inline composer.json]: to remove

File: composer.json [hunk 6]
```diff
@@ -5,49 +5,50 @@
...
+        "sylius-labs/coding-standard": "v4.1.1",
+        "symfony/browser-kit": "^4.4|^5.4",
+        "symfony/debug-bundle": "^4.4|^5.4|^6.0",
+        "symfony/dotenv": "^4.4|^5.4",
+        "symfony/intl": "^4.4|^5.4",
```
Comment (Jibbarth) [inline composer.json]: what the purpose of this lib here ?

File: composer.json [hunk 7]
```diff
@@ -5,49 +5,50 @@
...
-        "symfony/dotenv": "^4.3",
-        "symfony/intl": "^3.4|^4.3",
-        "symfony/lock": "^3.4|^4.3",
-        "symfony/web-profiler-bundle": "^3.4|^4.3",
-        "symfony/web-server-bundle": "^3.4|^4.3",
-        "webmozart/assert": "^1.6"
+        "sensiolabs/security-checker": "v6.0.3",
+        "sylius-labs/coding-standard": "v4.1.1",
+        "symfony/browser-kit": "^4.4|^5.4",
+        "symfony/debug-bundle": "^4.4|^5.4|^6.0",
```
Comment (Jibbarth) [inline composer.json]: Why only this one get the 6.0 version ? :upside_down_face:

File: ruleset/phpstan.neon [hunk 8]
```diff
@@ -1,10 +1,10 @@
 parameters:
-    level: max
+    level: 8
```
Comment (Jibbarth) [inline ruleset/phpstan.neon:2]: why not level max ?

File: src/Form/Type/MailTesterType.php [hunk 9]
```diff
@@ -43,7 +43,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             foreach ($options['data']['form_every_subjects'] as $subject) {
                 $builder->add(
                     $subject->getCode(),
-                    get_class($subject),
+                    (string) get_class($subject),
```
Comment (Jibbarth) [inline src/Form/Type/MailTesterType.php]: ?? Get_class always return a string, no ? https://www.php.net/manual/en/function.get-class.php

File: src/Form/Type/MailTesterType.php [hunk 10]
```diff
@@ -58,7 +58,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
 
         if (isset($options['data']['form_subject'])) {
             $builder
-                ->add('form_subject_chosen', get_class($options['data']['form_subject']), ['label' => false])
+                ->add('form_subject_chosen', (string) get_class($options['data']['form_subject']), ['label' => false])
```
Comment (Jibbarth) [inline src/Form/Type/MailTesterType.php]: same question :wink:

File: composer.json (hunk 5 above)
Comment (oallain) [reply composer.json]: I'll remove it, on all projets in the same time.

File: composer.json (hunk 7 above)
Comment (oallain) [reply composer.json]: Because version 5.4 allow php >=7.2.5 and version 6.0 allow php >= 8.0.2

File: composer.json (hunk 6 above)
Comment (oallain) [reply composer.json]: I remove it.

```
composer why symfony/intl
sylius/sylius-standard 1.11.x-dev requires (for development) symfony/intl (^4.4 || ^5.4)   
sylius/sylius          v1.11.6    requires                   symfony/intl (^4.4 || ^5.4)   
```

File: ruleset/phpstan.neon (hunk 8 above)
Comment (oallain) [reply ruleset/phpstan.neon:2]: Too many errors due of mixed types, i can't fix it here.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (oallain) [reply .github/workflows/ci.yaml]: Fixed

File: Makefile (hunk 2 above)
Comment (oallain) [reply Makefile]: I remove this line, Sylius have fixed it.

File: Makefile (hunk 3 above)
Comment (oallain) [reply Makefile]: Removerd

File: composer.json (hunk 4 above)
Comment (oallain) [reply composer.json]: I use new 1.8.0 version now 😄

File: composer.json (hunk 5 above)
Comment (oallain) [reply composer.json]: fixed

File: composer.json (hunk 7 above)
Comment (oallain) [reply composer.json]: My bad ... if removed it :)

**Review by TheGrimmChester — APPROVED** [review]

**Review by Jibbarth — APPROVED** [review]

**Pull request merged**